go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/bazelbuild/rules_go v0.23.2
 	github.com/btcsuite/btcd/btcec/v2 v2.3.2
 	github.com/consensys/gnark-crypto v0.12.1
+	github.com/crate-crypto/go-eth-kzg v1.1.0
```

Comment:
Note: go-eth-kzg is built ontop of go-kzg4844 -- once go-eth-kzg has been auditted, we can get rid of go-kzg4844 and use go-eth-kzg methods for 4844. Doing it this way means that its possible to merge in go-eth-kzg and it won't affect mainnet paths.